### PR TITLE
Update save_chat.py

### PR DESCRIPTION
**Pull request by soumyashaw**
## Issue that this pull request solves

Closes: #361 

## Proposed changes

Syntax Error while running

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: wplay/save_chat.py [hunk 1]
```diff
@@ -246,4 +246,4 @@ def runMain(mode, asset, bID):
             if len(drives) > 1:
                 print('Backup: ' + str(i))
                 folder = 'WhatsApp-' + str(i)
-            getMultipleFiles(drive[1], folder)
\ No newline at end of file
```
Comment (rpotter12) [inline wplay/save_chat.py:249]: Codacy Issue found: [String statement has no effect](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5598558)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Clones added
============
- wplay/save_chat.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5598558&bid=18341601)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Clones added
============
- wplay/save_chat.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5598558&bid=18341601)

File: wplay/save_chat.py [hunk 2]
```diff
@@ -246,4 +245,4 @@ def runMain(mode, asset, bID):
...
```
Comment (rpotter12) [inline wplay/save_chat.py:249]: Codacy Issue found: [String statement has no effect](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5598558)

**Comment by rpotter12** [comment]
@soumyashaw Resolve conflicts :)

**Comment by soumyashaw** [comment]
It was there when the pulled it. I thought it is important for future development. I just ensured it was error free and running

**Comment by rpotter12** [comment]
@soumyashaw No need of the commented portion.

**Pull request closed without merging**